specs/commandline/blob.md

```diff
@@ -29,11 +29,11 @@ Flags:
 Produce a detached signature for a given blob
 
 Usage:
-  notation blob sign [flags] <blob_path>
+  notation blob sign [flags] -n my-blob-signature <blob_path>
 
 Flags:
-  -s,  --signature path             output location of the detached signature
-       --media-type string          media type of the blob. If not provided, notation uses `application/octet-stream`
+  -n,  --signature-name string      friendly name for the detached signature. Signature file will be written to the currently working directory with this name plus ".sig" plus signature format as the file extension
```

Comment:
Can `--signature-name` be optional? Basically, the signature filename can be default to `<blob_path>.<type>`.

For example, `notation blob sign hello.tar` generates `hello.tar.jws` like `gzip hello.tar` generates `hello.tar.gz`.

Reply:
See https://github.com/notaryproject/notation/pull/811#discussion_r1384163309

Reply:
If users don't define a unique signature name, they might be hard to find signatures if there are multiple signatures generated in the same file path.

Reply:
@FeynmanZhou 

Per https://github.com/notaryproject/notation/pull/811#discussion_r1384262523, we are moving away from users choosing signature names.